qiita_db/meta_util.py

```diff
@@ -555,96 +555,144 @@ def generate_plugin_releases():
         f(redis_key, v)
 
 
-def get_software_commands(active):
-    software_list = [s for s in qdb.software.Software.iter(active=active)]
-    software_commands = defaultdict(lambda: defaultdict(list))
-
-    for software in software_list:
-        sname = software.name
-        sversion = software.version
-        commands = software.commands
-
-        for command in commands:
-            software_commands[sname][sversion].append(command.name)
-        software_commands[sname] = dict(software_commands[sname])
-
-    return dict(software_commands)
-
-
-def update_resource_allocation_redis(active=True):
+def update_resource_allocation_redis(active=True, verbose=False):
     """Updates redis with plots and information about current software.
 
     Parameters
     ----------
     active: boolean, optional
         Defaults to True. Should only be False when testing.
 
+    verbose: boolean, optional
+        Defaults to False. Prints status on what functin
+
     """
     time = datetime.now().strftime('%m-%d-%y')
-    scommands = get_software_commands(active)
+
+    # Retreave available col_name for commands
+    with qdb.sql_connection.TRN:
+        sql = 'SELECT col_name FROM qiita.resource_allocation_column_names;'
+        qdb.sql_connection.TRN.add(sql)
+        col_names_list = qdb.sql_connection.TRN.execute_fetchflatten()
```

Comment:
do not add to the variable name, the type ... this could be just col_names; however, I can see why you call it list ...